src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -678,17 +678,15 @@ private static class PileupQualBuffer {
         // as the *particular* alt base, whereas the qual actually means the probability of *any* substitution error.
         // since there are three possible substitutions for each ref base we must divide the error probability by three
         // which corresponds to adding 10*log10(3) = 4.77 ~ 5 to the qual.
-        private static final int ONE_THIRD_QUAL_CORRECTION = 5;
-
-        private static double USER_DEFINED_QUAL_CORRECTION = 1;
+        private static double MULTIPLE_SUBSTITUTION_BASE_QUAL_CORRECTION = 5;
```

Comment:
The constant default isn't needed anymore because now the default is set in the argument collection, right?